src/ApproximateGPs.jl

```diff
@@ -15,10 +15,10 @@ using KLDivergences
 
 using AbstractGPs: AbstractGP, FiniteGP, LatentFiniteGP, ApproxPosteriorGP, At_A, diag_At_A
 
-export SVGP, DefaultQuadrature, Analytic, GaussHermite, MonteCarlo
+export SparseVariationalApproximation, DefaultQuadrature, Analytic, GaussHermite, MonteCarlo
```

Comment:
Can you add a `@deprecate SVGP SparseVariationalApproximation` somewhere (and we might have to keep the export here)?